src/test/java/uk/ac/ic/wlgitbridge/application/WLGitBridgeIntegrationTest.java

```diff
@@ -727,6 +733,60 @@ public void usesCustomErrorHandler(
         wlgb.stop();
     }
 
+    @Test
+    public void cannotCloneAProtectedProject() throws IOException, GitAPIException, InterruptedException {
```

Comment:
This is somewhat confusing, because it is possible to clone a protected project; you just have to authenticate first. `cannotCloneAProtectedProjectWithoutAuthentication`?